packages/components/src/pinned-message/pinned-message.tsx

```diff
@@ -51,12 +66,22 @@ export const PinnedMessage = ({ messages }: PinnedMessageProps) => {
         </Pressable>
 
         <Wrapper>
-          <Text size={27} weight="semibold">
-            Pinned Messages
-          </Text>
-          {messages.map((message, i) => (
-            <Message key={i} {...message} />
-          ))}
+          <Button variant="grey" onPress={() => setIsDetailVisible(false)}>
+            &times;
+          </Button>
+          <DialogHeader>
+            <Text size={27} weight="semibold">
+              Pinned Messages
+            </Text>
+            <ContextTag icon="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.seadn.io%2Fgae%2FFG0QJ00fN3c_FWuPeUr9-T__iQl63j9hn5d6svW8UqOmia5zp3lKHPkJuHcvhZ0f_Pd6P2COo9tt9zVUvdPxG_9BBw%3Fw%3D500%26auto%3Dformat&f=1&nofb=1&ipt=c177cd71d8d0114080cfc6efd3f9e098ddaeb1b347919bd3089bf0aacb003b3e&ipo=images">
+              Rarible &rsaquo; # random
+            </ContextTag>
+          </DialogHeader>
+          <DialogContent>
+            {messages.map((message, i) => (
+              <Message key={i} {...message} />
```

Comment:
anti-pattern here, add ids as keys when we have it

Reply:
next time just add id's to the static data 😊

Reply:
Lol 🤦‍♂️